theforeman.org/yaml/jobs/test_2_5_stable.yaml

```diff
@@ -0,0 +1,40 @@
+- job:
+    name: test_2_5_stable
+    project-type: matrix
+    quiet-period: 2700
+    properties:
+      - github_foreman
+    scm:
+      - foreman:
+          branch: '2.5-stable'
+    triggers:
+      - scm_fifteen_minutes
+      - github
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+            - 2.5
+            - 2.6
```

Comment:
I think you can drop ruby 2.6 here, as we have no tergets using that.

@tbrisker was OK with that I think